templates/java/build-alt.gradle.mustache

```diff
@@ -11,7 +11,7 @@ apply plugin: 'signing'
 apply plugin: 'com.google.protobuf'
 
 description = 'GRPC library for service {{api.name}}-{{api.version}}'
-group = "io.gapi"
+group = "com.google.api"
```

Comment:
Well we are already using com.google.api for gax-java. We thought it might be easier to separate the Apiary world and vkit world if they were different.